src/content/docs/en/guides/integrations-guide/alpinejs.mdx

```diff
@@ -15,42 +16,77 @@ This **[Astro integration][astro-integration]** adds [Alpine.js](https://alpinej
 
 The `astro add` command-line tool automates the installation for you. Run one of the following commands in a new terminal window. (If you aren't sure which package manager you're using, run the first command.) Then, follow the prompts, and type "y" in the terminal (meaning "yes") for each one.
 
-```sh
-# Using NPM
-npx astro add alpinejs
-# Using Yarn
-yarn astro add alpinejs
-# Using PNPM
-pnpm astro add alpinejs
-```
+<PackageManagerTabs>
+  <Fragment slot="npm">
+    ```sh
+    npx astro add alpinejs
+    ```
+  </Fragment>
+  <Fragment slot="pnpm">
+    ```sh
+    pnpm astro add alpinejs
+    ```
+  </Fragment>
+  <Fragment slot="yarn">
+    ```sh
+    yarn astro add alpinejs
+    ```
+  </Fragment>
+</PackageManagerTabs>
 
 If you run into any issues, [feel free to report them to us on GitHub](https://github.com/withastro/astro/issues) and try the manual installation steps below.
 
```

Comment:
[suggestion/question]: This direction is a good start, but would'nt it be more helpful to encourage users to try the installation steps **before** raising an issue?  See my suggestion for more context. 
```suggestion
Try the manual installation steps below, and if you run into any issues, [feel free to report them to us on GitHub](https://github.com/withastro/astro/issues).

```